crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -760,7 +760,7 @@ impl<'w, 's> Commands<'w, 's> {
     /// watches those targets.
     ///
     /// [`Trigger`]: crate::observer::Trigger
-    pub fn trigger_targets(&mut self, event: impl Event, targets: impl TriggerTargets) {
+    pub fn trigger_targets(&mut self, event: impl Event, targets: impl TriggerTargets + 'static) {
```

Comment:
Why are you adding the `'static` bounds to the functions? Shouldn't it work without them?